pkg/networkservice/chains/nsmgr/server_heal_test.go

```diff
@@ -0,0 +1,371 @@
+// Copyright (c) 2020-2021 Doc.ai and/or its affiliates.
```

Comment:
Please add here some tests for the reconnecting to the same Endpoint/Forwarder.

Reply:
Done